jwk/helper_test.go

```diff
@@ -71,3 +71,23 @@ func TestIder(t *testing.T) {
 	assert.True(t, len(Ider("public", "")) > len("public:"))
 	assert.Equal(t, "public:foo", Ider("public", "foo"))
 }
+
+func TestHandlerFindPublicKey(t *testing.T) {
+	var testRSGenerator = RS256Generator{}
+	var testECDSAGenerator = ECDSA256Generator{}
+
+	t.Run("Test_Helper/Run_FindPublicKey_With_RSA", func(t *testing.T) {
+		RSIDKS, _ := testRSGenerator.Generate("test-id-1", "sig")
+		keys, err := FindPublicKey(RSIDKS)
+		require.NoError(t, err)
+		assert.Equal(t, keys.KeyID, Ider("public", "test-id-1"))
+
```

Comment:
One last thing, please ensure here that the key is indeed a public key:


```suggestion
		assert.IsType(t, keys.Key, new(rsa.PublicKey))
```